### ISSUE TITLE
Data does not sync between iPad and MacOS versions?

### ISSUE THREAD

**Issue by coreydaley** [design-first/system-designer#153]
**Describe the bug**
Systems created on the iPad version do not sync to the MacOS version, and systems created on the MacOS version do not sync to the iPad version. It would be nice if they would use iCloud Drive to sync data between the two.

**Comment by ecarriou** [comment]
For sync between iPad and MacOS you can use the sync with GitHub feature (see doc https://designfirst.io/systemdesigner/documentation/docs/en/sync-your-system-with-github.html). 
Is that resolved your issue ?

**Comment by coreydaley** [comment]
Yes, but it would be super convenient if it were to use iCloud storage like most other apps do. Any thoughts on providing that option?

**Comment by ecarriou** [comment]
A few years ago I have studied the subject but I thought that it added too much constraints for the users (at that time users should re-connect to iCloud if I use the iCloud JS API for ex). But I will have a look again to check if Apple eases now the process.
